src/main/java/games/strategy/triplea/settings/HasDefaults.java

```diff
@@ -1,5 +0,0 @@
-package games.strategy.triplea.settings;
```

Comment:
This interface was used to support the old settings framework. It was a brittle to get working and was pretty locked in to the existing settings UI structure.  I was never very happy with it, so the framework for this was rewritten, now we have all such configuration in one or two enums. It's still not super simple, but simpler and way more flexible than the previous framework.